configs/recognition/tin/tin_kinetics400-pretrained-tsm-r50_1x1x8-50e_kinetics400-rgb.py

```diff
@@ -99,3 +99,4 @@
 # optimizer
 optim_wrapper = dict(
     constructor='TSMOptimWrapperConstructor', paramwise_cfg=dict(fc_lr5=True))
+load_from = 'https://download.openmmlab.com/mmaction/recognition/tsm/tsm_r50_1x1x8_50e_kinetics400_rgb/tsm_r50_1x1x8_50e_kinetics400_rgb_20200607-af7fb746.pth'  # noqa: E501
```

Comment:
is it better to use the `pretrained` option like https://github.com/open-mmlab/mmaction2/blob/439bfcedc55562adada98997713c5257149974f5/configs/recognition/slowonly/slowonly_imagenet-pretrained-r50-nl-embedded-gaussian_8xb16-4x16x1-steplr-150e_kinetics400-rgb.py#L6

Reply:
@hukkai ok, i will modify it

Reply:
`pretrained` is used to load backbone ckpt, but `load form` is for the `recognizer`, and its ckpt has a prefix `backbone.`
another option is use `init_cfg`, which support remove prefix, but ResNetTIN pass the `init_weights` due to its weird initialization flow.
so we can only use `load_from`  here.